examples/06-verif-manual/vm-007-plastic_compression_of_a_pipe_assembly.py

```diff
@@ -0,0 +1,581 @@
+
+main_columns = {
+    "Target": target_res,
+    "Mechanical APDL": simulation_res,
+    "Ratio": list(np.divide(simulation_res, target_res))
+}
+
+row_tuple = [("PIPE288", "Load, lb for Deflection = 0.032 in"),
+             ("PIPE288", "Load, lb for Deflection = 0.05 in"),
+             ("PIPE288", "Load, lb for Deflection = 0.1 in"),
+
+             ("SOLID185", "Load, lb for Deflection = 0.032 in"),
+             ("SOLID185", "Load, lb for Deflection = 0.05 in"),
+             ("SOLID185", "Load, lb for Deflection = 0.1 in"),
+
+             ("SHELL181", "Load, lb for Deflection = 0.032 in"),
+             ("SHELL181", "Load, lb for Deflection = 0.05 in"),
+             ("SHELL181", "Load, lb for Deflection = 0.1 in")]
+
+index_names = ["Element Type", "Load Step"]
+row_indexing = pd.MultiIndex.from_tuples(row_tuple)
+df = pd.DataFrame(main_columns, index=row_indexing)
+
+df.style.set_caption('Results Comparison',
+                     ).set_table_styles([
+                        {
+                            "selector": "th.col_heading",
+                            "props": [("background-color", "#FFEFD5"),
+                                      ("color", "black"),
+                                      ("border", "0.5px solid black"),
+                                      ("font-style", "italic"),
+                                      ("text-align", "center")]
+                        },
+                        {
+                            "selector": "th.row_heading",
+                            "props": [("background-color", "#FFEFD5"),
+                                      ("color", "black"),
+                                      ("border", "0.5px solid black"),
+                                      ("font-style", "italic"),
+                                      ("text-align", "center")]
+                        },
+                        {
+                            "selector": "td:hover",
+                            "props": [("background-color", "#FFF8DC")]
+                        },
+                        {
+                            "selector": "th",
+                            "props": [("max-width", '120px')]
+                        },
+                        {
+                            "selector": "",
+                            "props": [('border', '0.5px solid black')]
+                        },
+                        {
+                            'selector': 'caption',
+                            'props': [('color', 'black'),
+                                      ("font-style", "italic"),
+                                      ('font-size', '24px'),
+                                      ("text-align", "center")]
+                        }],
+).set_properties(**{
+    "background-color": "#FFFAFA",
+    "color": "black",
+    "border-color": "black",
+    "border-width": "0.5px",
+    "border-style": "solid",
+    "text-align": "center"}).format("{:.3f}")
```

Comment:
I thought this was going to be shorted/removed.

Reply:
HI German(@germa89)
Alex told me that I can remain it for VM7.

Reply:
Good then. I didn't know that.